friends/src/components/Friend.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import axios from 'axios';
+
+import '../App.css';
+
+class Friend extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            age: '',
+            email: ''
+        }
+    }
+
+    inputChangehandler = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    deleteFriend = (e) => {
```

Comment:
I really like that you're using the deleteFriendFromList method in the parent component. You're making sure you only have 1 source of truth (this way of thinking really lends itself to Redux as well).